### ISSUE TITLE
Support Vega-Themes

### ISSUE THREAD

**Issue by domoritz** [vega/vega-embed#69]
https://github.com/vega/vega-themes

It would be nice to just say `{"theme": "dark"}` to get a dark chart. We still need to support config (which will override the theme).

**Comment by g3o2** [comment]
Maybe even let the option accept an array, allowing developers to build an "interactive" list box from it?

**Comment by domoritz** [comment]
Can you explain what you mean by an interactive list box?

**Comment by g3o2** [comment]
sorry for the imprecise terminology, I've meant a simple drop down list... (list box allows for multiple items to be selected, which makes no sense for this case)

**Comment by domoritz** [comment]
I see. Thanks for the clarification. I think that functionality could be provided outside of embed.

**Comment by g3o2** [comment]
Right, perhaps rather a nice feature for showing off / trying out the themes in the vega-editor.

**Comment by domoritz** [comment]
That's over at https://github.com/vega/editor/issues/58
